subprojects/zlib.wrap

```diff
@@ -0,0 +1,4 @@
+[wrap-git]
+directory=zlib
+url=https://github.com/centricular/zlib.git
+revision=meson
```

Comment:
There is [zlib wrap](https://wrapdb.mesonbuild.com/zlib) on Wrap DB, why do we need to use a fork here instead of it?

Reply:
There were several issues with the zlib wrap on WrapDB (windows build, header installation, .pc file, library versioning).

Pushing these things back into wrapdb is on our list, but these things take time and wrapdb is cumbersome. It's a requirement for GStreamer though so will definitely happen in the near future.

I can remove the wrap file (and the others pointing to non-upstream repos) if you prefer.